core/src/executor/evaluate/function.rs

```diff
@@ -236,6 +236,20 @@ pub fn abs(name: String, n: Evaluated<'_>) -> Result<Value> {
     }
 }
 
+pub fn ifnull(expr1: Evaluated<'_>, then: Evaluated<'_>) -> Result<Value> {
+    // should we pass expr2 as an Expr enum and evaluate it if expr1 is null?
+    // that would be more efficient than evaluating it before hand..
+    // what do you think?
```

Comment:
okay.. should we leave the comment in to remind us in the future, or just remove it for now?

Reply:
Except for doc comments which are for db users, I'm for having less comments as possible.
Code itself needs to be readable without depending on extra comments.
Maintaining comments are fully done by human, not compiler so it can be easily outdated.

I think we can deal with this issue in simple.
Both two options which we discussed are acceptable.
If this issues come again later, then we can think about this again.
If it doesn't, then we don't need to put effort on this.

Reply:
okay..   sounds good to me.
Sometimes I add comments (that are temporary), just to get advice or info from reviewers and I plan on deleting them before the code is merged into main.